package.json

```diff
@@ -35,8 +35,5 @@
     "rimraf": "5.0.1",
     "typescript": "^5.5.2",
     "vite": "^5.3.2"
-  },
```

Comment:
Hi @3200pro 👋 !  Thanks for catching this!

Perhaps we just update to match the [Astro requirements](https://docs.astro.build/en/tutorial/1-setup/1/#nodejs)?
```
{
  "engines": {
    "node": ">=18.17.1 <19.0.0 || >=20.3.0"
  }
}
```

Reply:
Working on that project now. I'll give this option a shot. One moment.

Reply:
Yeah, you're right.  I think fix works well.  Thanks!!